windows/src/ext/sentry/Sentry.Client.pas

```diff
@@ -222,34 +222,43 @@ procedure TSentryClient.DoTerminate;
   ExitProcess(1);
 end;
 
-procedure TSentryClient.DoBeforeEvent(const ExceptionClassName, Message: string;
+procedure TSentryClient.DoBeforeEvent(const EventID, ExceptionClassName, Message: string;
   EventType: TSentryClientEventType);
 var
   EventAction: TSentryClientEventAction;
 begin
   if Assigned(FOnBeforeEvent) then
   begin
     EventAction := sceaContinue;
-    FOnBeforeEvent(Self, EventType, ExceptionClassName, Message, EventAction);
+    FOnBeforeEvent(Self, EventType, EventID, ExceptionClassName, Message, EventAction);
     if EventAction = sceaTerminate then
       DoTerminate;
   end;
 end;
 
-procedure TSentryClient.DoAfterEvent(const ExceptionClassName, Message: string;
+procedure TSentryClient.DoAfterEvent(const EventID, ExceptionClassName, Message: string;
   EventType: TSentryClientEventType);
 var
 
  EventAction: TSentryClientEventAction;
```

Comment:
Should this be removed?

Reply:
Resolved -- whitespace is a lot cleaner now but it's made a big diff because of the EOL changes. Ignore whitespace when reviewing !